src/main/java/stepsdesign/NameSpaceadminSteps.java

```diff
@@ -0,0 +1,153 @@
+    CdfNameSpaceAdminActions.clickOnSavePreference();
+  }
+
+  @Then("Select a provisioner: {string} for the compute profile")
+  public void selectProvisionerForComputeProfile(String provisionerName) {
+    CdfNameSpaceAdminActions.selectProvisioner(provisionerName);
+  }
+
+  @Then("Click on close button of compute profile properties page")
+  public void closeButtonComputeProfile() {
+    CdfNameSpaceAdminActions.clickCloseButton();
+  }
+
+  @Then("Delete the preferences")
+  public void clickOnDeletePreference() {
+    CdfNameSpaceAdminActions.deletePreference();
+  }
+
+  @Then("Reset the preferences")
+  public void clickOnResetPreference() {
+    CdfNameSpaceAdminActions.clickOnResetPreference();
+  }
+
+  @Then("Verify the reset is successful for added preferences")
+  public void verifyIfResetValidatedSuccessfully() {
+    CdfNameSpaceAdminActions.verifyIfResetValidatedSuccessfully();
+  }
+
+  @Then("Click on: {string} button in the properties")
+  public void clickOnButtonInTheProperties(String buttonType) {
+    CdfNameSpaceAdminActions.clickCreateButtonComputeProfile(buttonType);
+  }
+
+  @Then("Verify that the compute profile is displaying an error message: {string} on the footer")
+  public void verifyErrorMessageDisplayedOnFooter(String errorMessageLocation) {
+    CdfNameSpaceAdminActions.verifyErrorMessageOnFooter(errorMessageLocation);
+  }
+
+  @Then("Enter the New Namespace Name with value: {string}")
+  public void enterNamespaceNameValue(String value) {
+    CdfNameSpaceAdminActions.enterNamespaceName(value);
+  }
+
+  @Then("Enter the Namespace Description with value: {string}")
+  public void enterNamespaceDescriptionValue(String value) {
+    CdfNameSpaceAdminActions.enterNamespaceDescription(value);
+  }
+
+  @Then("Verify the failed error message: {string} displayed on dialog box")
+  public void verifyFailedErrorMessageDisplayedOnDialogBox(String errorMessageLocation) {
+    CdfNameSpaceAdminActions.verifyFailedErrorMessageOnDialogBox(errorMessageLocation);
+  }
+
+  @Then("Switch to the newly created Namespace")
+  public void switchToNewNameSpace() {
+    CdfNameSpaceAdminActions.switchToNewNameSpace();
+  }
+
+  @Then("Verify if the switch is successful by checking the current {string} value")
+  public void verifyNameSpaceSwitch(String newNameSpaceName) {
+    CdfNameSpaceAdminActions.verifySwitchToNewNamespace(newNameSpaceName);
+  }
+
+  @Then("Add connection type as {string} and provide a {string}")
+  public void addConnectionInNameSpaceAdmin(String connectionType, String connectionName) {
+    CdfPluginPropertiesActions.clickPluginPropertyElement(connectionType);
+    CdfPluginPropertiesActions.enterValueInInputProperty("name", connectionName);
+    CdfPluginPropertiesActions.replaceValueInInputProperty("projectId", "projectId");
+  }
+
+  @Then("Verify that the user is navigated to nameSpace admin page successfully")
+  public void verifyThatHubPageIsSuccessfullyOpened() {
```

Comment:
Also the method name should be relevant with the step here the method says verify if hug page is opened but in step we are saying verify is user is navigated to name space admin page

Reply:
Yes , changed it to verifyThatNamespacePageIsSuccessfullyOpened()